.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,3 +15,12 @@
 - [Unit tests](https://scikit-image.org/docs/dev/development/contribute.html#testing)
 - A gallery example in `./doc/examples` for new features
 - [Contribution guide](https://scikit-image.org/docs/dev/development/contribute.html) is followed
+
+## Release note
+
+Summarize the introduced changes in the code block below in one or a few sentences. The
+summary will be included in the next release notes automatically:
+
+```release-note
+...
+```
```

Comment:
```suggestion
> **Release note:** ...
```
Another syntax suggestion for the summary feature. This one would have the advantage of automatic soft-wrapping and may make it more apparent that we expect a single paragraph here. But might be harder to parse. :shrug:

Reply:
The advantage of having it in triple back ticks is that you expect to do the formatting yourself, and it is clear exactly what is and isn't included (i.e. what you call the parsing). But I should also check whether you get a markdown rendered preview: probably only with the suggested syntax.

Reply:
We should discuss whether this section should be active by default; I feel like, in the vast majority of cases, we do not need release notes for PRs. So perhaps this needs to go into a comment?

Reply:
We could do that. Or we just move this section into our contribution guide and add a link. 

The reason for not making this a comment was to make it more visible. I see no harm in encouraging contributors to fill out this section even if it only a slightly longer version of the PR title, it won't be worse than the title and makes our release notes more useful without work on our site. 

It's probably something we need to collect experience with and re-evaluate during the next few releases...

Note, that the regex does only match if there is at least one alphanumeric character, so `...` isn't picked up.

Reply:
I really like how crisp the current release notes is, so I just want to make sure we only add release notes when they add value. A slightly expanded version of the title will make the release notes much longer, for little benefit.

That said, deleting is easier than adding, so also happy to try this out for a bit.

Reply:
Totally on board with this. Following this line of thinking, we could even condense some sections (e.g. "Maintenance") such that if the summary section is missing, we don't even show the title, just a PR number. 

Though I maintain that anything that requires user action or changes API, should contain enough information that users don't have to leave the release notes to understand the API change. I am sure we will figure out a good balance going forward. :)